Update payment-control.mdx (#406)
swan-io/docs: docs/developers/using-api/payment-control.mdx

- **Default response**: In case of a timeout, choose whether you **accept** or **reject** the transaction by default. Swan suggests choosing **accept** for a better cardholder experience.
- **Timeout**: Timeout is up to 10 seconds (10000 milliseconds) in Sandbox. In Live, it's 1.5 seconds (1500 milliseconds) and not configurable.
- **Secret**: Generate a secret. Make sure to add it to your `x-swan-secret` header.

> - **€0 Authorizations**: Enable zero-amount authorization forwarding. These are transactions merchants use to validate cards without charging them. Disabled by default.

Reviewer: This can be simplified - something like this? 
```suggestion
- **€0 authorizations**: Merchants can enable this to use zero-amount authorizations to check a card's validity. This is disabled by default.
```

> - **Credit Authorizations**: Enable credit authorization forwarding for refund-related transactions (identified by `operationType: "Credit"`). Disabled by default.

Reviewer: ```suggestion
- **Credit authorizations**: Enable credit authorization forwarding for refund-related transactions (identified by `operationType: "Credit"`). This is disabled by default.
```
